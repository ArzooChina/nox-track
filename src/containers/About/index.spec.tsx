import React from 'react';
import { render } from '@testing-library/react';

import Index from './index';

describe('Index Page', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Index />);
    expect(asFragment()).toMatchSnapshot();
  });
});
