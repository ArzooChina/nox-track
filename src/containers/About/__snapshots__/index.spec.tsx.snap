// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Page renders correctly 1`] = `
<DocumentFragment>
  <div
    class="m-0 flex flex-col w-screen h-screen text-white"
  >
    <header
      class="w-screen flex bg-gray-800"
    >
      <nav
        class="w-full flex justify-between"
      >
        <div
          class="p-4"
        >
          <a
            class="text-2xl uppercase"
            href="/"
          >
            <img
              alt="logo"
              src="/title.png"
            />
          </a>
        </div>
        <ul
          class="flex items-center justify-between"
        >
          <li
            class="px-4 uppercase hover:bg-gray-900 p-4 h-full leading-10 cursor-pointer"
          >
            <a>
              Login
            </a>
          </li>
          <li
            class="px-4 uppercase hover:bg-gray-900 p-4 h-full leading-10 cursor-pointer"
          >
            <a>
              About
            </a>
          </li>
        </ul>
      </nav>
    </header>
    <main>
      <h1
        class="text-xl text-center m-4"
        data-testid="helloH1"
      >
        Let's go.
      </h1>
    </main>
  </div>
</DocumentFragment>
`;
